src/components/ResultAction/ResultAction.ts

```diff
@@ -44,6 +42,10 @@ export abstract class ResultAction extends Component {
         tooltip: ComponentOptions.buildStringOption()
     };
 
+    public tooltip: HTMLElement;
```

Comment:
We shouldn't expose the tooltip element. It is breaking component encapsulation.